### PR TITLE
config: Use vanilla kernel and qemu for virtiofs.

### PR DESCRIPTION
**Pull request by jcvenegas**
Make default the usage of virtiofs with vanilla builds.

Depends-on: github.com/kata-containers/tests#3173
Depends-on: github.com/kata-containers/packaging#1190

Fixes: #3083

Signed-off-by: Carlos Venegas <jos.c.venegas.munoz@intel.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -276,7 +276,7 @@ ifneq (,$(QEMUVIRTIOFSCMD))
     # qemu-specific options (all should be suffixed by "_QEMU")
     DEFBLOCKSTORAGEDRIVER_QEMU_VIRTIOFS := virtio-fs
     DEFNETWORKMODEL_QEMU := tcfilter
-    KERNELNAMEVIRTIOFS = $(call MAKE_KERNEL_VIRTIOFS_NAME,$(KERNELTYPE))
+    KERNELNAMEVIRTIOFS = $(call MAKE_KERNEL_NAME,$(KERNELTYPE))
```
Comment (jodh-intel) [inline Makefile:279]: Not required for this PR, but we could do with some underscores to make some of these long variable names more readable ;)

**Review by jodh-intel — APPROVED** [review]
Thanks @jcvenegas.

**Review by likebreath — APPROVED** [review]

**Comment by cmaf** [comment]
I'm seeing this error in http://jenkins.katacontainers.io/job/kata-containers-runtime-vfio-PR/345/consoleText (and elsewhere):

```
ERROR: Docker-like container engine not accessible to current user:  (error exec: "podman": executable file not found in $PATH)
```

**Comment by jodh-intel** [comment]
The vfio CI failed in an interesting way @jcvenegas:

```
INFO: checking for container engine: podman
12:44:49 panic: ERROR: Docker-like container engine not accessible to current user:  (error exec: "podman": executable file not found in $PATH)
12:44:49 
12:44:49 goroutine 1 [running, locked to thread]:
12:44:49 github.com/kata-containers/runtime/cli.init.1()
12:44:49 	/home/jenkins/workspace/workspace/kata-containers-runtime-vfio-PR/go/src/github.com/kata-containers/runtime/cli/main_test.go:106 +0xfa8
12:44:49 FAIL	github.com/kata-containers/runtime/cli	0.173s
```

**Comment by jcvenegas** [comment]
Required jobs passing, adding do-not-merge label,  virtiofs metrics CI still is using experimental binaries and it is failing, but passing in the test repository this PR depends-on

**Pull request merged**